File: fix-db.mjs
import pg from 'pg'
const { Client } = pg

const client = new Client({ database: 'gulvisha', user: 'postgres', password: 'postgress', host: 'localhost' })
await client.connect()
const res = await client.query('UPDATE services SET display_order = 0 WHERE display_order IS NULL')
console.log('Updated:', res.rowCount)
await client.end()

File: fix-db.py
import psycopg2

conn = psycopg2.connect('dbname=gulvisha user=postgres password=postgress host=localhost')
cur = conn.cursor()
cur.execute('UPDATE services SET display_order = 0 WHERE display_order IS NULL')
conn.commit()
print('Updated', cur.rowcount, 'rows')
cur.execute('UPDATE services SET display_order = 0 WHERE display_order IS NULL')
conn.commit()
print('Verified:', cur.rowcount, 'nulls remaining')
conn.close()
